src/tests/stdlib/text_match_regex.ab

```diff
@@ -1,7 +1,12 @@
 import { match_regex } from "std/text"
 
 main {
-    if match_regex("Hello World", "World$") {
-        echo "Succeeded"
+    if not match_regex("Hello World", "World$") {
+        echo "\$ match failed"
```

Comment:
I know that this is not in scope of this issue, but the `$` should not be escaped in string. Everything that is in string literal (besides interpolation) should remain as is (excluding the special characters like `\n` etc)

Reply:
Thank you, it was because my muscle memory :)